engine/src/main/java/org/terasology/world/block/shapes/JsonBlockShapeLoader.java

```diff
@@ -120,7 +117,7 @@ public BlockShapeData deserialize(JsonElement json, Type typeOfT, JsonDeserializ
                 JsonObject collisionInfo = shapeObj.get(COLLISION).getAsJsonObject();
                 processCollision(context, shape, collisionInfo);
             } else {
-                shape.setCollisionShape(CUBE_SHAPE);
+                shape.setCollisionShape(PhysicsEngineManager.COLLISION_SHAPE_FACTORY.unitCube());
```

Comment:
Can the PhysicEngineManager qualifier be removed?

Reply:
Not directly, but I can introduce a class level constant that maps to the constant in the `PhysicsEngineManager`.

Reply:
You mean you can't write an import in the form:

import [...]PhysicsEngineManager.COLLISION_SHAPE_FACTORY

so that you can just write:

COLLISION_SHAPE_FACTORY.getUnitCube()

?

Reply:
Yes, that may be possible. I keep forgetting Java has static imports 😄